src/components/EditableTable.vue

```diff
@@ -9,7 +9,7 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 <template>
   <div class="editable-table">
     <table class="table" cellspacing="0">
-      <thead>
+      <thead cellspacing="0">
```

Comment:
does cellspacing=0 also need to be on the parent element?

Reply:
dude, I don't even know if it does anything at this point. I added this at a low point, when I was questioning my sanity. I'll try removing it and see if it changes anything.

Reply:
haha no worries either way, I don't think it's hurting anything but it just looked redundant when I was glancing over the code